modules/op-security-with-custom-scc.adoc

```diff
@@ -0,0 +1,17 @@
+[id='op-security-with-custom-scc_{context}']
+= Security with a custom SCC
+
+You can now leverage more security with custom SCC as it's easy to migrate pipelines that need to be run or built as root. However, if you would like to lock your installation further, you can only allow running the container as the user 1000 if the pod is asking for it and the default root is not allowed.
+
```

Comment:
```suggestion
. Procedure
```
- Write the following instructions as numbered steps. 
- The introduction implies that there are multiple steps, but the following topic shows only one step.
- Or, if there is only one step, start the line with a bullet point, `* `, rather than a number, `. `.